tests/testthat/test-unnest.R

```diff
@@ -154,21 +154,21 @@ test_that("unnest drops list cols if expanding", {
   df <- tibble(x = 1:2, y = list(3, 4), z = list(5, 6:7))
   out <- df %>% unnest(z)
 
-  expect_equal(names(out), c("x", "z"))
+  expect_named(out, c("x", "z"))
 })
 
 test_that("unnest keeps list cols if not expanding", {
   df <- tibble(x = 1:2, y = list(3, 4), z = list(5, 6:7))
   out <- df %>% unnest(y)
 
-  expect_equal(names(out), c("x", "z", "y"))
+  expect_equal(class(out$z), "list")
 })
 
 test_that("unnest respects .drop_lists", {
   df <- tibble(x = 1:2, y = list(3, 4), z = list(5, 6:7))
 
-  expect_equal(df %>% unnest(y, .drop = TRUE) %>% names(), c("x", "y"))
-  expect_equal(df %>% unnest(z, .drop = FALSE) %>% names(), c("x", "y", "z"))
+  expect_named(df %>% unnest(y, .drop = TRUE), c("x", "y"))
```

Comment:
Since there's now only a single `%>%`, I think you should just get rid of it too.